### DISCUSSION TITLE
Programming Help

### DISCUSSION THREAD

**Discussion by ZeaGmoN** [community/community#21510] [Other Feature Feedback, Questions, & Ideas]
<p>Hi, im student who’s a newbie at this. On the 1st class we got a task to make a snake game with letters in Visual Basic. (of course, the deadline is in 1 month) The catch is that snake eats blocks that contains each letter of ur name, so you go faster and faster untill u complete ur full name and i can’t find anything like that when im searching on the browser. After that it says Congrats and ur personal info, etc. My question is where can i learn to code that? </p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/zeagmon">@zeagmon</a>! Welcome to the community forum and congrats on starting your programming journey. As a first step, I suggest checking out this Doc from Microsft: <a href="https://docs.microsoft.com/en-us/visualstudio/get-started/visual-basic/tutorial-console" rel="nofollow noopener">https://docs.microsoft.com/en-us/visualstudio/get-started/visual-basic/tutorial-console</a> Please open another thread if you run into specific errors. I also suggest searching google for free Visual Basic online classes if you want additional learning materials. </p>
